common/istio-cni-1-24/README.md

```diff
@@ -1,7 +1,7 @@
 # Istio-CNI
 
-This uses istio-cni as described here https://istio.io/latest/docs/setup/additional-setup/cni/.
-Please be aware of Kserve and initcontainers https://istio.io/latest/docs/setup/additional-setup/cni/#compatibility-with-application-init-containers. Either use runasuser : 1337 xor add the annotation traffic.sidecar.istio.io/excludeOutboundIPRanges: 0.0.0.0/0 for kserve inferenceservices.
+This uses istio-cni as described here <https://istio.io/latest/docs/setup/additional-setup/cni/>.
+Please be aware of Kserve and initcontainers <https://istio.io/latest/docs/setup/additional-setup/cni/#compatibility-with-application-init-containers>. Either use runasuser : 1337 xor add the annotation traffic.sidecar.istio.io/excludeOutboundIPRanges: 0.0.0.0/0 for kserve inferenceservices.
```

Comment:
If you enable native sidecars for istio and istio-cni this might help to mitigate it already.

Reply:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: istiod
  namespace: istio-system
spec:
  template:
    spec:
      containers:
      - env:
        - name: ENABLE_NATIVE_SIDECARS
          value: "true"
        name: discovery
```
but maybe there is a more elegant way to configure it in the profile directly